.github/workflows/publish-apple-darwin-wasm.yml

```diff
@@ -74,6 +74,8 @@ jobs:
         run: |
           npm install
           npm test
+        env:
+          NARGO_PATH: "./noir/dist/nargo"

```

Comment:
Is there a reason we want to use this relative path install? I think we could do `cargo install` instead of `cargo build` and then use a globally installed nargo command, right?

Reply:
Not sure how I feel about that. It would definitely be cleaner for running the tests but it feels "wrong" when the purpose of this workflow it to build nargo in order to package it up to attach to the release. A little bit of the tail wagging the dog.

Reply:
I agree with you, but I've also encountered "relocation" bugs where something can be run from the target/output directory but crashes when moved somewhere else. If my assumption is correct, `cargo install` builds to the target directory and then moves the binary into the cargo bin path, so we'd be testing that relocation works. Is that correct?

Reply:
That's true. We're less likely to have hardcoded paths, etc. which only work from the `.cargo/bin` compared to relative to the build directory.